lib/LaTeXML/Package/TeX.pool.ltxml

```diff
@@ -99,7 +99,19 @@ RequireResource('LaTeXML.css');
 
 DefParameterType('Plain', sub {
     my ($gullet, $inner) = @_;
-    my $value = $gullet->readArg;
+    my $value = $gullet->readArg();
+    if ($inner) {
+      ($value) = $inner->reparseArgument($gullet, $value); }
+    $value; },
+  reversion => sub {
+    my ($arg, $inner) = @_;
+    (T_BEGIN,
+      ($inner ? $inner->revertArguments($arg) : Revert($arg)),
+      T_END); });
+
+DefParameterType('DefPlain', sub {
```

Comment:
I could technically add the masks treatment to `Plain` and `Expanded` directly and still have all tests work, but I thought this is a hair more efficient in performance. Since the masks are calculated and carried around only when requested, not requesting them in `Plain` and `Expanded` speeds things along a small amount.